docs/getting-started/server/guide.md

```diff
@@ -231,6 +231,14 @@ done by double-clicking on the downloaded certificate.
 5. The dev.pfx file will ask for a password. You can get this by clicking and opening the Licensing
    Certificate - Dev item in the vault
 
+:::note
```

Comment:
❓ Should this be a note, or info maybe? I'm not really sure how I feel about it.